setup.py

```diff
@@ -6,13 +6,14 @@
 
 setup(
     name="Open edX",
-    version="0.4",
+    version="0.5",
```

Comment:
I'm surprised this didn't cause requirements to get re-installed.  One possible reason may be that in devstack we only update requirements when particular files change.

It's probably worth updating the [paver script](https://github.com/edx/edx-platform/blob/master/pavelib/prereqs.py) to automatically re-install prerequisites when this `setup.py` file changes.  Otherwise, merging this PR could break a lot of developer environments.